projects/demo/src/modules/components/block-status/block-status.template.html

```diff
@@ -43,6 +43,7 @@
         <tui-doc-demo>
             <tui-block-status [card]="card">
                 <img
+                    tuiSlot="top"
```

Comment:
the `Setup` should before the `API` section

<img width="427" alt="image" src="https://user-images.githubusercontent.com/12021443/236158784-f2871d12-f6a0-4841-8eb4-78a7366f0fd5.png">

Reply:
why?

Reply:
oh, sorry, my fault